src/AutoRest.CSharp/Common/Generation/Writers/ClientOptionsWriter.cs

```diff
@@ -10,7 +10,7 @@ namespace AutoRest.CSharp.Generation.Writers
 {
     internal class ClientOptionsWriter
     {
-        public static void WriteClientOptions(CodeWriter writer, ClientOptionsTypeProvider clientOptions)
+        public static void WriteClientOptions(CodeWriter writer, ClientOptionsTypeProvider clientOptions, bool suppressNestedClientSpans)
```

Comment:
Can we have the default value of 'false' so we don't need to change the code in DataPlaneTarget.cs